src/main/java/mate/academy/DataProcessingException.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String massage, Throwable cause) {
```

Comment:
There is a typo in the parameter name 'massage'. It should be 'message' to correctly represent the error message being passed to the exception.